package.json

```diff
@@ -17,22 +23,53 @@
     "url": "https://github.com/eface2face/mimemessage.js/issues"
   },
   "dependencies": {
-    "debug": "^3.1.0",
-    "random-string": "^0.2.0"
+    "debug": "^2.2.0",
+    "random-string": "^0.1.2"
   },
   "devDependencies": {
-    "browserify": "^16.2.2",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-external-helpers": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-decorators": "^7.0.0",
+    "@babel/plugin-proposal-do-expressions": "^7.0.0",
+    "@babel/plugin-proposal-export-default-from": "^7.0.0",
+    "@babel/plugin-proposal-export-namespace-from": "^7.0.0",
+    "@babel/plugin-proposal-function-bind": "^7.0.0",
+    "@babel/plugin-proposal-function-sent": "^7.0.0",
+    "@babel/plugin-proposal-json-strings": "^7.0.0",
+    "@babel/plugin-proposal-logical-assignment-operators": "^7.0.0",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.0.0",
+    "@babel/plugin-proposal-numeric-separator": "^7.0.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/plugin-proposal-optional-chaining": "^7.0.0",
+    "@babel/plugin-proposal-pipeline-operator": "^7.0.0",
+    "@babel/plugin-proposal-throw-expressions": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-syntax-import-meta": "^7.0.0",
+    "@babel/plugin-transform-regenerator": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.0.0",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "babel-loader": "^8.0.4",
+    "babel-plugin-istanbul": "^4.1.4",
+    "eslint": "^5.7.0",
+    "eslint-config-airbnb-base": "^13.1.0",
+    "eslint-plugin-import": "^2.14.0",
     "expect.js": "^0.3.1",
-    "gulp": "^4.0.0",
+    "gulp": "git+https://github.com/gulpjs/gulp.git#4.0",
     "gulp-filelog": "^0.4.1",
-    "gulp-header": "^2.0.5",
-    "gulp-jscs": "^4.1.0",
-    "gulp-jscs-stylish": "^1.4.0",
-    "gulp-jshint": "^2.1.0",
-    "gulp-mocha": "^5.0.0",
-    "jshint": "^2.9.5",
-    "jshint-stylish": "^2.2.1",
-    "vinyl-source-stream": "^2.0.0"
+    "gulp-header": "^1.7.1",
+    "gulp-jscs": "^3.0.2",
+    "gulp-jscs-stylish": "^1.2.1",
+    "gulp-jshint": "^1.12.0",
+    "gulp-mocha": "^2.1.3",
+    "husky": "^1.1.2",
```

Comment:
You added husky but I don't see its config with prettier ? cf https://prettier.io/docs/en/precommit.html